File: uikit-sample/src/main/java/com/sendbird/uikit_messaging_android/utils/PushUtils.java
package com.sendbird.uikit_messaging_android.utils;

import androidx.annotation.NonNull;

import com.sendbird.android.SendBirdPushHandler;
import com.sendbird.android.SendBirdPushHelper;

public class PushUtils {

    public static void registerPushHandler(@NonNull SendBirdPushHandler handler) {
        SendBirdPushHelper.registerPushHandler(handler);
    }

    public static void unregisterPushHandler(@NonNull SendBirdPushHelper.OnPushRequestCompleteListener listener) {
        SendBirdPushHelper.unregisterPushHandler(listener);
    }
}

File: uikit-sample/src/main/kotlin/com/sendbird/uikit_messaging_android/utils/PushUtils.kt
package com.sendbird.uikit_messaging_android.utils

import com.sendbird.android.handler.PushRequestCompleteHandler
import com.sendbird.android.push.SendbirdPushHandler
import com.sendbird.android.push.SendbirdPushHelper

/**
 * This provides methods to manage push handler.
 */
object PushUtils {
    fun registerPushHandler(handler: SendbirdPushHandler) {
        SendbirdPushHelper.registerPushHandler(handler)
    }

    fun unregisterPushHandler(listener: PushRequestCompleteHandler) {
        SendbirdPushHelper.unregisterPushHandler(listener)
    }
}
